spec/services/hyrax/custom_queries/navigators/find_files_spec.rb

```diff
@@ -22,78 +26,93 @@
 
   describe '#find_original_file' do
     context 'when original file exists' do
-      let!(:original_file) { FactoryBot.create_using_test_adapter(:hyrax_file_metadata) }
-      let!(:fileset) { FactoryBot.create_using_test_adapter(:hyrax_file_set, files: [original_file], original_file: original_file) }
+      let(:original_file) { FactoryBot.valkyrie_create(:hyrax_file_metadata) }
+      let(:fileset) { FactoryBot.valkyrie_create(:hyrax_file_set, files: [original_file]) }
+
       it 'returns file metadata resource' do
         expect(query_handler.find_original_file(file_set: fileset).id.to_s).to eq original_file.id.to_s
       end
     end
 
     context 'when files do not exist' do
-      let!(:fileset) { FactoryBot.build(:hyrax_file_set) }
+      let(:fileset) { FactoryBot.build(:hyrax_file_set) }
+
       it 'raises error' do
         expect { query_handler.find_original_file(file_set: fileset) }
-          .to raise_error ::Valkyrie::Persistence::ObjectNotFoundError, "File set's original file is blank"
+          .to raise_error ::Valkyrie::Persistence::ObjectNotFoundError
```

Comment:
I like removing the second parameter to the `raise_error`, as that should be evident from the spec context.

Reply:
:+1:  it seemed like it was making the test fragile.